src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate begin = LocalDate.parse(dateFrom, DATE_FORMAT);
+        LocalDate end = LocalDate.parse(dateTo, DATE_FORMAT);
+        StringBuilder salaryInformation = new StringBuilder();
+        salaryInformation.append("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo);
```

Comment:
why do we write each method from the new line?

Reply:
https://mate-academy.github.io/style-guides/java/java.html#s4.5-line-wrapping
https://mate-academy.github.io/style-guides/java/java.html#s4.5.1-line-wrapping-where-to-break
А именно:
Note: While the typical reason for line-wrapping is to avoid overflowing the column limit, even code that would in fact fit within the column limit may be line-wrapped at the author's discretion

Указывая операторы так, я не думал о том чтоб "меньше сделать", а сделать код более читабельным.